macros/spec2.ejs

```diff
@@ -177,6 +177,7 @@ var status = {
   'Manifest'                   : 'WD',
   'MathML2'                    : 'REC',
   'MathML3'                    : 'REC',
+  'Media Capabilities'         : 'Living',
```

Comment:
I think the WICG drafts have been classified as 'ED' so far. Can you explain why this is a living standard now? (I'm fine with it, I just want to learn why this is 'Living' now.)

Reply:
Neither the phrase "living standard" nor "editor's draft" actually appears in the text of the spec. I picked the one that isn't an official w3c term.

Reply:
Honestly, I don't have strong feelings about it.

Reply:
@chrisdavidmills or @teoli2003, thoughts on how to classify WICG specs generally? I think we've put 'ED' so far. I have no strong feelings either, would like to be somewhat consistent about it, though.

Reply:
I'd classify them as "Draft" — if you look at the spec in question — https://wicg.github.io/media-capabilities/ — it is titled "Media Capabilities Draft Community Group Report, 15 June 2017".

Which sounds like a draft to me.

Also, if you read Marcos' blog post about the WICG process:

https://www.w3.org/blog/2015/07/wicg/

You'll see that in the Process section it states that WICGs are for conceptualizing ideas; only after you have got buy-in for your idea, will the spec be moved to an appropriate W3C WG. Which implies that before then it will only ever be in draft status.

I could be wrong of course, but this makes sense to me from what I've read.

Reply:
"Draft" would be the most appropriate description. I just noticed something. The macro processes abbreviations not listed in the macro instructions (317 to 414). 

I'm changing it to 'Draft' I'll submit a separate PR to update the instructions.